src/service/growthService.js

```
@@ -8,6 +8,22 @@ class GrowthService {
         throw error;
     }
   }
+
+  static async getDop() {
+    try {
+      return axiosInstance.get("/growth/dop");
+    } catch (error) {
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)